Formula/yadm.rb

```diff
@@ -0,0 +1,15 @@
+class Yadm < Formula
+  desc "Yet Another Dotfiles Manager"
+  homepage "https://github.com/TheLocehiliosan/yadm"
+  url "https://github.com/TheLocehiliosan/yadm/archive/1.04.tar.gz"
+  sha256 "a73aa51245866ce67aeb4322a62995ebbb13f29dc35508f486819dceb534968a"
+
+  def install
+    bin.install "yadm"
+    man1.install "yadm.1"
+  end
+
+  test do
+    system "yadm", "version"
```

Comment:
Sure. yadm actually has a large set of unit and acceptance tests, but they aren't included here because the tests have more dependencies than yadm itself (for example [bats](https://github.com/sstephenson/bats) and gnupg).

Does it seem reasonable if I update the tests to have yadm initialize a repo, set some of its own configurations, and confirm the repo and configs?